lib/tools/system-calls.js

```diff
@@ -546,14 +546,24 @@ systemCallMethods.killAllEmulators = async function () {
  * @param {string} avdName - The name of the emulator to be killed.
```

Comment:
return description is missing. Also it worth to mention about the argument is optional

Reply:
This is why I hate docstrings.